graphql/e2e/common/mutation.go

```diff
@@ -6082,3 +6087,207 @@ func updateLangTagFields(t *testing.T) {
+				" SportsMember which implements same interface",
+			query: `mutation addLibraryMember($input: [AddLibraryMemberInput!]!) {
+                       addLibraryMember(input: $input, upsert: false) {
+                         libraryMember {
+                           refID
+                         }
+                       }
+                     }`,
+			variables: `{
+                         "input": {
+                             "refID": "102",
+                             "name": "James",
+                             "itemsIssued": [
+                                 "Intro to C"
+                             ],
+                             "readHours": "1d2hr"
+                         }
+                     }`,
+			error: "couldn't rewrite mutation addLibraryMember because failed to rewrite mutation" +
+				" payload because interface Member; field refID: id 102 already exists for one of" +
+				" the implementing type of interface",
+		}, {
+			name: "adding new Cricket Team with upsert returns returns error as given id already exist" +
+				" in other node of type SportsMember which implements same interface",
+			query: `mutation addCricketTeam($input: [AddCricketTeamInput!]!) {
+                      addCricketTeam(input: $input, upsert: true) {
+                        cricketTeam {
+                          teamID
+                        }
+                      }
+                    }`,
+			variables: `{
+                         "input": {
+                             "teamID": "T01",
+                             "teamName": "Slash",
+                             "numOfBatsmen": 5,
+                             "numOfBowlers": 4
+                         }
+                     }`,
+			error: "couldn't rewrite mutation addCricketTeam because failed to rewrite mutation" +
+				" payload because interface Team; field teamID: id T01 already exists for" +
+				" one of the implementing type of interface",
+		}, {
+			name: "adding new Library manager returns error when it try to links to LibraryMember" +
+				" but got id of some other implementing type which implements " +
+				"same interface as LibraryMember",
+			query: `mutation addLibraryManager($input: [AddLibraryManagerInput!]!) {
+                      addLibraryManager(input: $input, upsert: false) {
+                        libraryManager {
+                          name
+                        }
+                      }
+                    }`,
+			variables: `{
+                          "input": {
+                              "name": "John",
+                              "manages": {
+                                  "refID": "102"
+                              }
+                          }
+                       }`,
+			error: "couldn't rewrite mutation addLibraryManager because failed to rewrite mutation" +
+				" payload because interface Member; field refID: id 102 already exists for one" +
+				" of the implementing type of interface",
```

Comment:
This error could be confusing. I think in this error case we are sure that this is some other implementing type. Can we make that clear in the error message.